### PR TITLE
Ppc64le

### PR DESCRIPTION
**Pull request by nehaljwani**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by nehaljwani** [comment]
Blocked on https://github.com/conda-forge/openldap-feedstock/pull/5

**Comment by nehaljwani** [comment]
@conda-forge-admin restart ci

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: .scripts/build_steps.sh [hunk 1]
```diff
@@ -29,12 +29,17 @@ source run_conda_forge_build_setup
 # make the build number clobber
 make_build_number "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 
+if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
+     EXTRA_CB_OPTIONS="${EXTRA_CB_OPTIONS:-} --no-test"
```
Comment (isuruf) [inline .scripts/build_steps.sh:33]: @nehaljwani, this should work now

**Pull request merged**